src/js/tabs/firmware_flasher.js

```diff
@@ -65,7 +65,19 @@ TABS.firmware_flasher.initialize = function (callback) {
 
             self.enableFlashing(true);
 
-            $('div.release_info .target').text(TABS.firmware_flasher.selectedBoard);
+            let targetName = TABS.firmware_flasher.selectedBoard;
+            let targetRegexp = /^(?:([^-]{1,4})-)?(.*)$/;
+            let targetParts = targetName.match(targetRegexp);
+            if (targetParts) {
+                targetName = targetParts[2];
+                if (targetParts[1]) {
+                    $('div.release_info #manufacturerInfo').show();
+                    $('div.release_info .manufacturer').text(targetParts[1]);
```

Comment:
your span with the class of `manufacturer` is inside the `#manufacturerInfo`.

I think you should use id for both.

```html
                <div id="manufacturerSection">
                    <strong i18n="firmwareFlasherReleaseManufacturer"></strong>
                    <span id="manufacturerInfo"></span>
                    <br />
                </div>
```

Reply:
Good point - was copy / pasta. I suspect the original author of a lot of this was allergic to ids.